src/core/Akka/Actor/Scheduler/HashedWheelTimerScheduler.cs

```diff
@@ -159,11 +158,15 @@ private void Run()
                 _startTime = 1;
             }
 
+            // we've already started - need to prevent duplicates.
+            if (_workerInitialized.IsSet)
```

Comment:
Still need to guarantee single-threaded nature even if we don't have a single dedicated thread to process anymore.